index.css

```diff
@@ -0,0 +1,32 @@
+body {
+    display: flex;
+    flex-direction: column;
+align-items: center;
+background-color: lavender;
+font-family: monospace;
+}
+header {
+  width: 70%;
+  height: 150px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+background-color: skyblue;
+margin-bottom:30px ;
+padding: 10px;
+}
```

Comment:
Don't forget to use prettier, you don't want you tech lead to read PR's like this